install-panel.sh

```diff
@@ -874,6 +845,22 @@ function main {
     [ -z "$MYSQL_PASSWORD" ] && print_error "MySQL password cannot be empty"
   done
 
+  print_brake 88
+  echo "* Please follow the steps below. The installer will ask you for configuration details."
+  print_brake 88
+  echo ""
+
+  echo "* Choose your timezone. e.g. [Europe/Amsterdam]: " # This is really hard without autocomplete
```

Comment:
Yes, no autocomplete is a problem really. Is there anything that can be done? I believe Pterodactyl uses the [PHP standard format for timezones](https://www.php.net/manual/en/timezones.php). Perhaps we can manually verify that the timezone is valid somehow?